Sources/ApolloCodegenLib/Templates/LegacyAPQOperationManifestTemplate.swift

```diff
@@ -3,26 +3,20 @@ import Foundation
 /// Provides the format to output an operation manifest file used for APQ registration.
 struct LegacyAPQOperationManifestTemplate: OperationManifestTemplate {
 
-  struct OperationJSONValue: Codable {
-    let name: String
-    let source: String
+  func render(operations: [OperationManifestItem]) -> String {
+    template(operations).description
   }
 
-  func render(operations: [OperationManifestItem]) throws -> String {
-    try template(operations).description
-  }
-
-  private func template(_ operations: [OperationManifestItem]) throws -> TemplateString {
-    let encoder = JSONEncoder()
```

Comment:
So the JSON encoding was causing issues with us now trying to actually have a `\n` in our strings before a fragment, so the persisted query manifest would would fine because it manually creates it's JSON but this would fail tests because the encoding was removing our newline characters.